webpack/assets/javascripts/react_app/components/Layout/layout.scss

```diff
@@ -87,6 +87,14 @@
       }
     }
   }
+
+  .visible-xs-block {
+    display: none !important;
+
+    @media (max-width: $pf-global--breakpoint--lg) {
+      display: block !important;
+    }
+  }
```

Comment:
can you avoid using `!important`?

Reply:
Changed the class name to not clash with bootstrap